src/js/setUpSite.js

```diff
@@ -236,11 +238,26 @@ const setUpSite = async () => {
   setUpAbout();
 
   const map = createMap();
-  await Promise.all([setUpCitiesLayer(map), setUpParkingLotsLayer(map)]);
+  await Promise.all([setUpCitiesLayer(map), setUpParkingLotsLayer(map), setUpSearch(initialCityId, "atlanta-ga")]);
 
   // There have been some issues on Safari with the map only rendering the top 20%
   // on the first page load. This is meant to address that.
   map.invalidateSize();
 };
 
+const setUpSearch = (initialCityId, fallbackCityId) => {
+  cities = {};
```

Comment:
Why have this? It's unused

Reply:
I agree. I removed this in my newest commit!